data-prepper-plugins/otel-metrics-source/src/test/java/org/opensearch/dataprepper/plugins/source/otelmetrics/OTelMetricsSourceTest.java

```diff
@@ -200,6 +201,7 @@ public void beforeEach() {
         pipelineDescription = mock(PipelineDescription.class);
         when(pipelineDescription.getPipelineName()).thenReturn(TEST_PIPELINE_NAME);
         SOURCE = new OTelMetricsSource(oTelMetricsSourceConfig, pluginMetrics, pluginFactory, pipelineDescription);
+        assertTrue(SOURCE.getDecoder() instanceof OTelMetricDecoder);
```

Comment:
This would be better:

```
assertThat(SOURCE.getDecoder(), instanceOf(OTelMetricDecoder.class));
```

If this test were to fail you would clearly see what the actual class is.

The same for the similar lines.